smrf/envphys/radiation.py

```diff
@@ -1076,3 +1205,360 @@ def get_hrrr_cloud(df_solar, df_meta, logger, lat, lon):
+        [9.50e-1, 6.0174026672672744e-1],
+        [1.00e+0, 5.3760268141094450e-1],
+        [1.10e+0, 4.2662303940385946e-1],
+        [1.20e+0, 3.4879316059369590e-1],
+        [1.30e+0, 2.8537622230393302e-1],
+        [1.40e+0, 2.4213740074273104e-1],
+        [1.50e+0, 2.0682569646774943e-1],
+        [1.60e+0, 1.7583787434888801e-1],
+        [1.70e+0, 1.4557069925604664e-1],
+        [1.80e+0, 1.1458287713718522e-1],
+        [1.90e+0, 9.0801525278524142e-2],
+        [2.00e+0, 7.4226643680063386e-2],
+        [2.10e+0, 6.4858232341802959e-2],
+        [2.20e+0, 5.6931115055582595e-2],
+        [2.30e+0, 4.9003997769362232e-2],
+        [2.40e+0, 4.6121409665282103e-2],
+        [2.50e+0, 3.8914939405081775e-2],
+        [2.60e+0, 3.4591057248961578e-2],
+        [2.70e+0, 3.0987822118861412e-2],
+        [2.80e+0, 2.8105234014781281e-2],
+        [2.90e+0, 2.5222645910701150e-2],
+        [3.00e+0, 2.2340057806621019e-2],
+        [3.10e+0, 1.8736822676520856e-2],
+        [3.20e+0, 1.6286622788052742e-2],
+        [3.30e+0, 1.3836422899584630e-2],
+        [3.40e+0, 1.1962740631932545e-2],
+        [3.50e+0, 1.0521446579892480e-2],
+        [3.60e+0, 9.7287348512704438e-3],
+        [3.70e+0, 8.8639584200464042e-3],
+        [3.80e+0, 7.9991819888223649e-3],
+        [3.90e+0, 7.4226643680063384e-3],
+        [4.00e+0, 6.8461467471903120e-3],
+        [4.10e+0, 6.2696291263742857e-3],
+        [4.20e+0, 5.6210468029562563e-3],
+        [4.30e+0, 5.1165938847422333e-3],
+        [4.40e+0, 4.6842056691302139e-3],
+        [4.50e+0, 4.2518174535181938e-3],
+        [4.60e+0, 3.8194292379061740e-3],
+        [4.70e+0, 3.4591057248961576e-3],
+        [4.80e+0, 3.2429116170901479e-3],
+        [4.90e+0, 2.9546528066821346e-3],
+        [5.00e+0, 2.7600781096567258e-3],
+        [6.00e+0, 1.2611322955350576e-3],
+        [7.00e+0, 7.1344055575983254e-4],
+        [8.00e+0, 4.3238821561201970e-4],
+        [9.00e+0, 2.7384586988761247e-4],
+        [1.00e+1, 1.8016175650500822e-4],
+        [1.10e+1, 1.2250999442340559e-4],
+        [1.20e+1, 8.6477643122403945e-5],
+        [1.30e+1, 6.2696291263742856e-5],
+        [1.40e+1, 3.9635586431101808e-5],
+        [1.50e+1, 3.5311704274981611e-5],
+        [1.60e+1, 2.7384586988761248e-5],
+        [1.70e+1, 2.2340057806621018e-5],
+        [1.80e+1, 1.7295528624480788e-5],
+        [1.90e+1, 1.4412940520400656e-5],
+        [2.00e+1, 1.1530352416320527e-5],
+        [2.50e+1, 4.3959468587222005e-6],
+        [3.00e+1, 2.1619410780600985e-6],
+        [3.50e+1, 1.1530352416320526e-6],
+        [4.00e+1, 6.7740820445883089e-7],
+        [5.00e+1, 2.7384586988761249e-7],
+        [6.00e+1, 1.3692293494380625e-7],
+        [8.00e+1, 5.0445291821402299e-8],
+        [1.00e+2, 2.1619410780600986e-8],
+        [1.00e+3, 0.0000000000000000e+0]])
+
+#####################################################################
+# IPW function
+#####################################################################
```

Comment:
So how will these legacy IPW wrappers be used? Will these just remain for the short-term so we can compare the functions by using the tests?

Reply:
Leaving them in doesn't hurt at the moment and are mainly used in the test comparisons. Perhaps we should leave them in for a while until we're comfortable with the performance of the new functions.